test/configTestCases.test.js

```diff
@@ -47,13 +48,13 @@ describe('ConfigTestCases', function () {
               return done(err);
             }
 
-            function _it(title, fn) {
+            function testIt(title, fn) {
```

Comment:
`eslint-config-airbnb@^8.0.0` and up forbid dangling `_`, so I came up with slightly more descriptive names...